GOVERNANCE_DOC.md

```diff
@@ -0,0 +1,591 @@
+- `PlainTextProposal`. All the proposals that do not involve a modification of the source code go under this type. For example, an opinion poll would use a proposal of type `PlainTextProposal`
+- `SoftwareUpgradeProposal`. If accepted, validators are expected to update their software in accordance with the proposal. They must do so by following a 2-steps process described in the [Software Upgrade](#software-upgrade) section below. Software upgrade roadmap may be discussed and agreed on via `PlainTextProposals`, but actual software upgrades must be performed via `SoftwareUpgradeProposals`.
+
+### Proposal categories
+
+There are two categories of proposal:
+- `Regular`
+- `Urgent`
+
+These two categories are strictly identical except that `Urgent` proposals can be accepted faster if a certain condition is met. For more information, see [Threshold](#threshold) section.
+
+
+## Vote
+
+### Participants
+
+*Participants* are users that have the right to vote. On the Cosmos Hub, participants are bonded Atom holders. Unbonded Atom holders and other users do not get the right to participate in governance. However, they can submit and deposit on proposals.
+
+### Voting period
+
+Once a proposal reaches `MinDeposit`, it immediately enters `Voting period`. We define `Voting period` as the interval between the moment the vote opens and the moment the vote closes. `Voting period` should always be shorter than `Unbonding period` to prevent double voting. The initial value of `Voting period` is 2 weeks.
+
+### Option set
+
+The option set of a proposal refers to the set of choices a participant can choose from when casting its vote.
+
+The initial option set includes the following options: 
+- `Yes`
+- `No`
+- `NoWithVeto` 
+- `Abstain` 
+
+`NoWithVeto` counts as `No` but also adds a `Veto` vote. `Abstain` option allows voters to signal that they do not intend to vote in favor or against the proposal but accept the result of the vote. 
+
+*Note: from the UI, for urgent proposals we should maybe add a ‘Not Urgent’ option that casts a `NoWithVeto` vote.*
+
+### Quorum 
+
+Quorum is defined as the minimum percentage of voting power that needs to be casted on a proposal for the result to be valid. 
+
+In the initial version of the governance module, there will be no quorum enforced by the protocol. Participation is ensured via the combination of inheritance and validator's punishment for non-voting.
+
+### Threshold
+
+Threshold is defined as the minimum ratio of `Yes` votes to `No` votes for the proposal to be accepted.
+
+Initially, the threshold is set at 50% with a possibility to veto if more than 1/3rd of votes (excluding `Abstain` votes) are `NoWithVeto` votes. This means that proposals are accepted is the ratio of `Yes` votes to `No` votes at the end of the voting period is superior to 50% and if the number of `NoWithVeto` votes is inferior to 1/3rd of total votes (excluding `Abstain`).
```

Comment:
language needs to be fixed. If threshold is minimum ratio of Yes:No and threshold is 50%, means 1:2...

So either it's "minimum proportion of Yes votes" or some other fixed description but I don't think its the ratio of Yes to No.

The same language needs to be fixed in a variety of places